packages/terra-popup/src/terra-dev-site/doc/common/PopupCommonExample.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import Button from 'terra-button';
 import PropTypes from 'prop-types';
-import Popup from 'terra-popup';
 import Placeholder from 'terra-doc-template/lib/Placeholder';
+import Popup from '../../../Popup';
```

Comment:
Why was this changed?

Reply:
Oops..! I didn't notice this Thank you for pointing out. Removed